app/controllers/appointments_controller.rb

```diff
@@ -74,6 +78,6 @@ def set_appointment
 
   # Only allow a list of trusted parameters through.
   def appointment_params
-    params.require(:appointment).permit(:title, :description, :start_time, :end_time)
+    params.require(:appointment).permit(:title, :description, :start_time, :end_time, :selected_date)
```

Comment:
Given your `selected_date` parameter is not inside the `appointment` parameter, you can remove it from here.
```suggestion
    params.require(:appointment).permit(:title, :description, :start_time, :end_time)
```